samples/mixed/consumer/src/main.rs

```diff
@@ -0,0 +1,283 @@
+fn start_send_notification_repeater(provider_uri: String, consumer_uri: String) {
+    info!("Starting the Consumer's send notification repeater.");
+    tokio::spawn(async move {
+        loop {
+            // info!("Invoking the send_notification command on endpoint {}", &provider_uri);
+
+            let client_result = ProviderClient::connect(provider_uri.clone()).await;
+            if client_result.is_err() {
+                continue;
+            }
+            let mut client = client_result.unwrap();
+
+            let response_id = Uuid::new_v4().to_string();
+
+            let payload: String = String::from("The send_notification request.");
+
+            let request = tonic::Request::new(InvokeRequest {
+                entity_id: String::from(SEND_NOTIFICATION_COMMAND_ID),
+                consumer_uri: consumer_uri.clone(),
+                response_id,
+                payload,
+            });
+
+            let _response = client.invoke(request).await;
+
+            sleep(Duration::from_secs(5)).await;
+        }
+    });
+}
+
+/// Start the set ui message repeater.
+///
+/// # Arguments
+/// `provider_uri` - The provider_uri.
+/// `consumer_uri` - The consumer_uri.
+fn start_set_ui_message_repeater(provider_uri: String, consumer_uri: String) {
+    info!("Starting the Consumer's send notification repeater.");
+    tokio::spawn(async move {
+        let array: [&str; 10] =
+            ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"];
+        let mut index: usize = 0;
+        loop {
+            // info!("Invoking the set_ui_message command on endpoint {}", &provider_uri);
+
+            let client_result = ProviderClient::connect(provider_uri.clone()).await;
+            if client_result.is_err() {
+                continue;
+            }
+            let mut client = client_result.unwrap();
+
+            let response_id = Uuid::new_v4().to_string();
+
+            let request = tonic::Request::new(InvokeRequest {
+                entity_id: String::from(SET_UI_MESSAGE_COMMAND_ID),
+                consumer_uri: consumer_uri.clone(),
+                response_id,
+                payload: array[index].to_string(),
+            });
+
+            let _response = client.invoke(request).await;
+
+            index = (index + 1) % array.len();
+
+            sleep(Duration::from_secs(5)).await;
+        }
+    });
+}
+
+/// Start the set ui message repeater.
+///
+/// # Arguments
+/// `provider_uri` - The provider_uri.
+/// `consumer_uri` - The consumer_uri.
+fn start_activate_air_conditioning_repeater(provider_uri: String, consumer_uri: String) {
+    info!("Starting the Consumer's send notification repeater.");
+    tokio::spawn(async move {
+        let mut is_active = true;
+        loop {
+            // info!("Invoking the set_ui_message command on endpoint {}", &provider_uri);
+
+            let client_result = ProviderClient::connect(provider_uri.clone()).await;
```

Comment:
same as above - do you need to reconnect every time?
Is it ok to swallow errors?